conf/kyuubi-env.sh.template

```diff
@@ -38,6 +38,7 @@
 # - KYUUBI_WORK_DIR_ROOT    Root directory for launching sql engine applications.
 #                           (Default: $KYUUBI_HOME/work)
 # - HADOOP_CONF_DIR         Directory containing the Hadoop / YARN configuration to use.
+# - YARN_CONF_DIR         Directory containing the YARN configuration to use.
```

Comment:
what are the precedences between these, will they fail some engines if we propagate them together to the engine side?

Reply:
The classpath is `${HADOOP_CLASSPATH}:${HADOOP_CONF_DIR}:${YARN_CONF_DIR}` is Flink. Therefore this will not fails.